bindings/go/evmc/evmc.go

```diff
@@ -15,6 +15,8 @@ package evmc
 #include <stdlib.h>
 #include <string.h>
 
+#include "loader/loader.c"
```

Comment:
I think this should be a problem because the C comment in go files are compiled twice if any Go function is exported. I think it only works because there is nothing exported here.
I may have modified version of this, will try to push it there.

Reply:
It misses some definitions w/o that import for some reason... At least on a mac.

Reply:
I changed that.